lib/config/config_test.go

```diff
@@ -68,6 +68,7 @@ func TestDefaultValues(t *testing.T) {
 		OverwriteRemoteDevNames: false,
 		TempIndexMinBlocks:      10,
 		UnackedNotificationIDs:  []string{},
+		DismissedPanels:         []string{},
```

Comment:
I think you could just use `UnackedNotificationIDs` for this. It's admittedly not it's original purpose, but I'd personally still rather stretch its scope than add a new config field just for this panel.

Reply:
Or use the same mechanism that UnackedNotificationIDs  uses to show a one-off modal like we do for other stuff, just change the display condition to have the additional constraints on addresses and other stuff.